restorer.go

```diff
@@ -19,57 +19,31 @@ type Restorer struct {
 	Filter func(item string, dstpath string, node *Node) bool
 }
 
-// NewRestorer creates a restorer preloaded with the content from the snapshot snid.
-func NewRestorer(s *server.Server, snid backend.ID) (*Restorer, error) {
-	r := &Restorer{s: s}
+var abortOnAllErrors = func(str string, node *Node, err error) error { return err }
```

Comment:
I think having it in a variable like this is nicer than adding a comment (also there was a bug: since this was defined as a closure, it was referencing the `err` from the surrounding context, not the error that was being passed into the function (it wasn't even named))